tests/test_ecl_output.cc

```diff
@@ -246,6 +254,7 @@ void test_readWriteWells()
     CHECK( wellRatesCopy.get( "OP_1" , opt::wat) , wellRates.get( "OP_1" , opt::wat));
     CHECK( wellRatesCopy.get( "OP_2" , 188 , opt::wat) , wellRates.get( "OP_2" , 188 , opt::wat));
 }
+}
```

Comment:
This looks weird? I think it closes the anonymous namespace? If so, please add `// anonymous namespace`.

Other than that I think this is fine and agree it can be merged.

Reply:
> This looks weird?

Agreed.

> I think it closes the anonymous namespace?

It does.  I'll add the suggested annotation.